src/pudl/output/eia860.py

```diff
@@ -136,11 +138,168 @@ def plants_eia860(pudl_engine, start_date=None, end_date=None):
     out_df = (
         pd.merge(out_df, utils_eia_df, how="left", on=["utility_id_eia"])
         .dropna(subset=["report_date", "plant_id_eia"])
+        .pipe(fill_in_missing_ba_codes)
         .pipe(apply_pudl_dtypes, group="eia")
     )
     return out_df
 
 
+def make_consistent_ba_code_column(plants: pd.DataFrame) -> pd.DataFrame:
+    """Make a columns of  the most consistent balancing authority code.
+
+    Employ the harvesting function :func:`occurrence_consistency` which determines how
+    consistent the values in a table are across all records within each plant. This
+    function grabs only the values determined to be at least 70% consitent and merges
+    them onto the plants table as a new column: ``balancing_authority_code_eia_consistent``
+    """
+    ba_code_consistent = occurrence_consistency(
+        entity_idx=["plant_id_eia"],
+        compiled_df=plants,
+        col="balancing_authority_code_eia",
+        cols_to_consit=["plant_id_eia"],
+        strictness=0.7,
+    )
+
+    static_plant_to_code_map = ba_code_consistent[
+        ba_code_consistent.balancing_authority_code_eia_consistent
```

Comment:
oh this is confusing... the `balancing_authority_code_eia_consistent` column is a boolean column that comes out of `occurrence_consistency` to let you know if the value in the output has met the strictness test. But then we effectively rename the most consistent BA code the same name after the bool column is dropped. I'll fix name collision this somehow.